packages/message-forwarder/src/server.e2e.auto.test.ts

```diff
@@ -28,13 +28,15 @@ const resubmittedAho = fs.readFileSync("src/test/fixtures/success-exceptions-aho
 describe("Server in auto mode", () => {
   let messageData: string
   let correlationId: string
+  let mf: MessageForwarder
```

Comment:
A bit of nitpick, but I'd prefer to be explicit even though we're in the context of the message forwarder package, just to remove that smol cognitive load. It means `await messageForwarder.start()` reads nicer too.

```suggestion
  let messageForwarder: MessageForwarder
```

Reply:
That's fair - will change now